batch/batch/server/server.py

```diff
@@ -690,9 +690,10 @@ class Batch:
     MAX_TTL = 30 * 60
 
     @staticmethod
-    def from_record(record):
+    def from_record(record, deleted=False):
         if record is not None:
-            assert not record['deleted']
+            if not deleted:
```

Comment:
AFAIK were preferring deleted != False

Reply:
I think pylint will complain that it should be simplified.